### ISSUE TITLE
arc flow panic on "negative repeat count"

### ISSUE THREAD

**Issue by michaeljs1990** [pinkluz/arcanist#1]
```panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat(0xb14e20, 0x1, 0xfffffffffffffffa, 0xc002b53458, 0x12)
        /home/mschuett/go/src/strings/strings.go:529 +0x5e5
github.com/pinkluz/arcanist/lib/console.gloss(0xc002b6da08, 0x8, 0x8, 0xfffffffffffffffa, 0x1, 0xb14e20, 0x1, 0xe34860)
        /home/mschuett/arcanist/lib/console/color.go:92 +0xbc7
github.com/pinkluz/arcanist/lib/console.drawLine(0xc002b3af00, 0xc00018e500, 0x1, 0xe74b10, 0x0, 0x0, 0x1, 0x1, 0xc002b6db40)
        /home/mschuett/arcanist/lib/console/draw_graph.go:181 +0x6cd
github.com/pinkluz/arcanist/lib/console.walkNodes(0xc002b6dc67, 0xc00018e500, 0x1, 0xe74b10, 0x0, 0x0, 0x1, 0x0, 0x14, 0xc002b6dc48, ...)
        /home/mschuett/arcanist/lib/console/draw_graph.go:73 +0xda
github.com/pinkluz/arcanist/lib/console.walkNodes(0xc002b6dc67, 0xc00018e000, 0x0, 0xc002b6dc60, 0x0, 0x0, 0xc000021d00, 0x1, 0x93e440, 0xc000021d60, ...)
        /home/mschuett/arcanist/lib/console/draw_graph.go:84 +0x291
github.com/pinkluz/arcanist/lib/console.DrawGraph(0xc003977160, 0x1, 0x1, 0xc0001871a0, 0x0, 0x0, 0x2)
        /home/mschuett/arcanist/lib/console/draw_graph.go:49 +0x14e
github.com/pinkluz/arcanist/cli/flow.(*flowCmd).run(0xc0000105e0, 0xc000192f00, 0xe74b10, 0x0, 0x0)
        /home/mschuett/arcanist/cli/flow/flow.go:32 +0x31a
github.com/spf13/cobra.(*Command).execute(0xc000192f00, 0xe74b10, 0x0, 0x0, 0xc000192f00, 0xe74b10)
        /home/mschuett/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc000129400, 0x0, 0xffffffff, 0xc00002e0b8)
        /home/mschuett/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/mschuett/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
        /home/mschuett/arcanist/cmd/arc/main.go:11 +0x30
```

**Comment by michaeljs1990** [comment]
This is an issue with long branch names. I need to handle this better and adjust the padding based on the length of the longest branch name.

**Comment by michaeljs1990** [comment]
Fixed in commits 3cb237b and 96cc780.
